assets/management/commands/deleteSampleData.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from assets import models
+
+
+class Command(BaseCommand):
+    help = 'Deletes testing sample data'
+
+    def handle(self, *args, **kwargs):
+        from django.conf import settings
+
+        if not (settings.DEBUG):
+            raise CommandError('You cannot run this command in production')
+
+        # self.delete_categories()
+        # self.create_statuses()
+        # self.create_suppliers()
+        # self.create_collections()
+        # self.create_assets()
```

Comment:
Are these commented bits required, or can they be tidied up?

Reply:
I have removed them, again leftover by PyAssets